src/main/java/edu/cornell/kfs/concur/ConcurParameterConstants.java

```diff
@@ -10,5 +10,14 @@
     public static final String CONCUR_CUSTOMER_PROFILE_SUB_UNIT = "CONCUR_CUSTOMER_PROFILE_SUB_UNIT";
     public static final String DEFAULT_TRAVEL_REQUEST_OBJECT_CODE = "DEFAULT_TRAVEL_REQUEST_OBJECT_CODE";
     public static final String CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE = "CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE";
-
+    public static final String CONCUR_SAE_COLLECTOR_DOCUMENT_TYPE = "CONCUR_SAE_COLLECTOR_DOCUMENT_TYPE";
+    public static final String CONCUR_SAE_COLLECTOR_SYSTEM_ORIGINATION_CODE = "CONCUR_SAE_COLLECTOR_SYSTEM_ORIGINATION_CODE";
```

Comment:
As per the specs, the SAE-to-PDP origination code should be "Z6", but the SAE-to-Collector origination code should be "CN" instead. Thus, we do need a separate parameter for this.